recipes/sbepp/all/conanfile.py

```diff
@@ -37,7 +37,7 @@ def _min_cppstd(self):
     def _compilers_minimum_version(self):
         if self.options.with_sbeppc:
             return {
-                "gcc": "9",
+                "gcc": "8.1",
```

Comment:
```suggestion
                "gcc": "8",
```
You will have to use additional logic to not exclude compiler.version=8 otherwise:
```python
def loose_lt_semver(v1, v2):
    return all(int(p1) < int(p2) for p1, p2 in zip(str(v1).split("."), str(v2).split(".")))
```

Reply:
I did not test gcc8. Most probably it works too, but it is not available on the ftp.